versions.yaml

```diff
@@ -78,6 +90,27 @@ versions:
       redis:
         sha256: 6624841267e142c5d5d5be292d705f8fb6070677687c5aad1645421a936d22b3
         version: '5.0.6'
+    excludeTests:
+    - tests/functional_tests/redis_exporter_test.yaml
 
+
```

Comment:
nit: rm double blank line, one is enough.

Reply:
Fixed,.